cdk/l3-cdk-resources/atlas-private-endpoint/.projenrc.js

```diff
@@ -0,0 +1,51 @@
+const { awscdk, JsonFile } = require('projen');
+const project = new awscdk.AwsCdkConstructLibrary({
+    author: 'MongoDBAtlas',
+    authorAddress: 'https://mongodb.com',
+    authorName: 'MongoDBAtlas',
+    cdkVersion: '2.1.0',
+    defaultReleaseBranch: 'main',
+    name: '@mongodbatlas-awscdk/atlas-basic-private-endpoint',
+    repositoryUrl: 'https://github.com/mongodb/mongodbatlas-cloudformation-resources.git',
+    stability: 'experimental',
+    peerDeps: [
+        '@mongodbatlas-awscdk/atlas-basic',
+        '@mongodbatlas-awscdk/private-endpoint'
+    ],
+    devDeps: ['aws-cdk', 'ts-node'],
+    sampleCode: false,
+    keywords: ['cdk',
+        'awscdk',
+        'aws-cdk',
+        'cloudformation',
+        'cfn',
+        'extensions',
+        'constructs',
+        'cfn-resources',
+        'cloudformation-registry',
+        'l3',
+        'mongodb',
+        'atlas',
+        'cluster',
+        'dbuser',
+        'ip-access-list'],
+    description: 'Creates a project, cluster, dbuser, ipaccess list and private endpoint in MongoDB Atlas',
```

Comment:
[small change] you cannot create an ipaccesslist but you add an entry to that list

```suggestion
    description: 'Creates a project, cluster, dbuser, private endpoint and add an IP entry to the access list in MongoDB Atlas',
```